docs/source/reference/pdisplay.rst

```diff
@@ -47,27 +52,46 @@ Then, add ``pcvl.pdisplay()`` of your circuit.
 Also, you can change the display of the circuit using a different skin which can itself be configured.
 Indeed, a boolean can be set to obtain a more compact display (if the circuit is too wide for example).
 
->>> import perceval as pcvl
->>> import perceval.components.unitary_components as comp
->>> from perceval.rendering.circuit import SymbSkin
->>> C = pcvl.Circuit.decomposition(pcvl.Matrix(comp.PERM([3, 1, 0, 2]).U),
->>> comp.BS(R=pcvl.P("R")), phase_shifter_fn=comp.PS)
->>> symbolic_skin = SymbSkin(compact_display=True)
->>> pcvl.pdisplay(C, skin=symbolic_skin)
+.. code-block::
+
+  import perceval as pcvl
+  import perceval.components.unitary_components as comp
+  from perceval.rendering.circuit import SymbSkin
+
+  C = pcvl.Circuit.decomposition(pcvl.Matrix(comp.PERM([3, 1, 0, 2]).U),
+  comp.BS(R=pcvl.P("R")), phase_shifter_fn=comp.PS)
+  symbolic_skin = SymbSkin(compact_display=True)
+  pcvl.pdisplay(C, skin=symbolic_skin)
 
 .. figure:: ../_static/img/decomposition_symb_compact.png
   :align: center
   :width: 25%
 
->>> symbolic_skin = SymbSkin(compact_display=False)
->>> pcvl.pdisplay(C, skin=symbolic_skin)
+.. code-block::
+
+  symbolic_skin = SymbSkin(compact_display=False)
+  pcvl.pdisplay(C, skin=symbolic_skin)
 
 .. figure:: ../_static/img/decomposition_symb_compact_false.png
   :align: center
   :width: 25%
 
+By default the skin will be ``PhysSkin``, if you want to use another skin by default, you can save your configuration into your Perceval persistent configuration.
+
+In order to do that you need to use the object ``DisplayConfig`` like that:
+
+.. code-block::
+
+  from perceval.rendering.circuit import DisplayConfig, SymbSkin
```

Comment:
Should we expose `DisplayConfig` and skin classes directly from `perceval.rendering` to simplify this import? (I'm for it)